Rakefile

```diff
@@ -64,28 +66,45 @@ namespace :docs do
       f.puts("---")
       f.puts
       f.puts(documentation.base_docstring)
+      f.puts
+      f.puts("## Examples")
+      f.puts
+
+      initialize_method = documentation.meths.find(&:constructor?)
+
+      initialize_method.tags(:example).each do |tag|
+        f.puts("### #{tag.name}")
+        f.puts
+
+        # rubocop:disable Security/Eval
+        html = render_inline(eval(tag.text.gsub("<%= render(", "").gsub(") %>", ""))).to_html
```

Comment:
This rake task is only run locally, so this `eval` should be OK here.

Reply:
Do we think we'll ever run this via actions? Is there any kind of safety we could add here to ensure we don't run it in that environment?